### ISSUE TITLE
[todo] BERT loss on MSA

### ISSUE THREAD

**Issue by lucidrains** [lucidrains/alphafold2#80]
needs to be added

**Comment by lucidrains** [comment]
need to make noise accurate to the paper (contiguous spans?)

**Comment by lucidrains** [comment]
<img width="647" alt="Screen Shot 2021-07-23 at 2 08 53 PM" src="https://user-images.githubusercontent.com/108653/126841788-9dab15a8-bce1-4458-9ce3-59099bc75887.png">

**Comment by lucidrains** [comment]
used at inference time.. 🤔

**Comment by intersun** [comment]
> used at inference time.. 🤔

Yep, I also noticed this part, which seems really weird to me...
